xep-0321.xml

```diff
@@ -55,7 +55,7 @@
   <p>This document addresses the following requirements:</p>
   <ol>
     <li>Make it possible for remote services or entities to manage user's roster by the same mechanisms that descibed in the &rfc6121;.</li>
-    <li>Provide a way for user to control which services have permission to manage his roster.</li>
+    <li>Provide a way for user to control which services have permission to manage their roster.</li>
```

Comment:
`for user` should be either `for the user` or `for users`. I personally think plural is better, in which case to above sentence also needs to be changed to plural, otherwise it should be `a user's roster`.